docs/concepts/index.md

```diff
@@ -5,7 +5,10 @@ The Concepts section of the Kubernetes documentation is a work in progress.
 
 #### Object Metadata
 
-[Annotations](/docs/concepts/object-metadata/annotations/)
+
+* [Annotations](/docs/concepts/object-metadata/annotations/)
+* [Stateful Sets](/docs/concepts/controllers/statefulsets/)
```

Comment:
I'm not sure that I follow. The link works. This is the correct location of the asset. Did you mean something else?